docs/source/pages/howToWorkWithPlatform/adminApplication/users/users.rst

```diff
@@ -2,173 +2,191 @@
 
-.. image:: pic_users/email.png
-   :width: 600
-   :align: center
+Single corporate user creation
+==============================
 
-2. Follow the link to create password to user profile
+As an administrator of an account you can create a corporate user, which is useful for company accounts. To do so follow next steps:
 
-.. image:: pic_users/emailFollowLink.png
-   :width: 600
+1. Navigate to https://platform_address_here/admin/users
+2. Click the "Create user" button and select "Create user" option from the menu
+3. Fill in all fields on the user creation form. Also you can choose should a personal mailbox be created for this user
+
+.. image:: pic_users/createUserModal.png
+   :width: 400
    :align: center
 
-3. Set password and confirm it
+4. Click the "Create" button
+5. After this steps user will be created and you will be redirected to the permissions configuration page
+6. You can also open Settings tab to edit user details, such as name, timezone, phone (if it is enabled for users in instance settings), default mailbox etc.
 
-.. image:: pic_users/createPassword.png
-   :width: 600
+.. image:: pic_users/settingsTab.png
+   :width: 400
    :align: center
 
-4. Authorize on platform using credentials above
-5. Now user can use platform according to the permissions were set
+.. note:: If you are editing existing users, this tab will be available only for corporate users.
 
-Create any quantity of users
-============================
+7. When all permissions and settings are defined click the "Save" button in the header of the page
 
-As administrator of account you can create users with verified and activated domain. It can be used if you working in company and you have to create corporate users. How to add, verify and activate domain you can find by link :ref:`domains configuration <domains>`. To create more than one user you can click the "Create" button in header and select respective option "Create users". Modal window appeares and you will be able to download the example of the file for massive user creation. You can open the file using any tool which support .xlsx format. Also, you have to save the file in this particular format, so don't change extention.
+Mass corporate users creation
+=============================
 
-What is included in the report
-==============================
+As an administrator of account you also can mass-create corporate users, which is useful for company accounts with a big number of employees. To do so, click the "Create user" button in the page header and select "Create user" option from the menu. A form will appear, allowing you to download the example file for mass user creation. You can open the file using any tool which support .xlsx format. Also, you have to save the file in this particular format, so make sure to not change the file extention after editing.
```

Comment:
To do so, click the "Create user" button in the page header and select "Create user"

To do so, click the "Create user" button in the page header and select "Create users"